docs/explainability.md

```diff
@@ -45,6 +45,8 @@ article](https://en.wikipedia.org/wiki/Shapley_value) on Shapley values or
 Lloyd Shapley's [original
 paper](https://www.rand.org/content/dam/rand/pubs/research_memoranda/2008/RM670.pdf).
 
+**NOTE: Tree depth is limited to 32 for shapley value computation. Tree models with higher depth will throw an error.**
```

Comment:
Only for CPU execution, right? If so, could we update accordingly?

Reply:
Also just to confirm, since I wasn't immediately able to find the corresponding logic: We do catch the error and rethrow as a `rapids::TritonException`, right? If so, could we just update the docs here with the error type we expect?

Reply:
Wait! I just found the logic, and yep we're doing it exactly right. So just mentioning that it is specifically an `Unsupported` error would be nice to help folks with debugging.